src/collectors/collector_worker.go

```diff
@@ -80,60 +80,70 @@ func (cw *CollectorWorker) RunCollectors() {
 // Run the collector for this device.
 func (cw *CollectorWorker) Run(device configuration.DeviceConfig) (string, error) {
 	log.Printf("Collect Start: %s\n", device.Name)
-	var connection *ssh.Client
+	var lastResult string
 
 	// call buildCollector to assemble the collector and batcher
 	collector, batchSlice, err := cw.buildCollector(device)
 	if err != nil {
 		return "", err
 	}
 
-	// Set up device SSHConfig
-	s := cw.buildSSH(collector, device)
+	// route53 does not use ssh
+	// TODO: This should be fixed - quick solution to skip SSH for a single collector, but type doesn't scale
+	if device.Type != "route53" {
```

Comment:
Maybe check if device.SSHAuthMethod is empty or something. (Similar to my other comment on device specific references in the main stream of things.